mainnet/jobs.json

```diff
@@ -78,5 +78,45 @@
     "name": "ethusd_cryptocompare",
     "selector": "[USD]",
     "url": "https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD"
+  },
+  { 
+    "weight": 1,
+    "power": 2,
```

Comment:
Can you update power for all these SKALE jobs to `6` for better precision
@manuelbarbas